parquet/src/main/java/org/apache/iceberg/parquet/ReadConf.java

```diff
@@ -185,7 +186,12 @@ private Map<Long, Long> generateOffsetToStartPos(Schema schema) {
       return null;
     }
 
-    try (ParquetFileReader fileReader = newReader(file, ParquetReadOptions.builder().build())) {
+    FileDecryptionProperties decryptionProperties =
+        (options == null) ? null : options.getDecryptionProperties();
+
+    try (ParquetFileReader fileReader =
+        newReader(
+            file, ParquetReadOptions.builder().withDecryption(decryptionProperties).build())) {
```

Comment:
Style: it would be better to create the new read options on a separate line.

Reply:
I tried a couple of ways to break the line(s), but the spotless check hasn't accepted them,  the spotless apply always returns the code to this form.

Reply:
I'm not talking about reformatting the line breaks, I mean to create a variable:

```java
ParquetReadOptions readOptions = ParquetReadOptions.builder().withDecryption(decryptionProperties).build();

try (ParquetFileReader fileReader = newReader(file, readOptions)) {
  ...
}
```